src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -25,18 +38,57 @@ class JavaScriptSdkLoader(BaseView):
     def determine_active_organization(self, request: Request, organization_slug=None) -> None:
         pass
 
-    def _get_context(self, key):
+    def _get_bundle_kind_modifier(self, key: ProjectKey, sdk_version: str) -> Tuple[str, bool]:
+        """Returns a string that is used to modify the bundle name"""
+
+        # If we load a performance/replay bundle, set this to be false, as we do not want to
+        # load the default bundle.
+        isLazy = True
+        bundle_kind_modifier = ""
+
+        # The order in which these modifiers are added is important, as the
+        # bundle name is built up from left to right.
+        # https://docs.sentry.io/platforms/javascript/install/cdn/
+
+        if get_dynamic_sdk_loader_option(key, DynamicSdkLoaderOption.HAS_PERFORMANCE):
+            bundle_kind_modifier += ".tracing"
+            isLazy = False
+
+        has_replay = get_dynamic_sdk_loader_option(key, DynamicSdkLoaderOption.HAS_REPLAY)
+
+        if has_replay:
+            bundle_kind_modifier += ".replay"
+            isLazy = False
+
+        # From JavaScript SDK version 7 onwards, the default bundle code is ES6, however, in the loader we
+        # want to provide the ES5 version. This is why we need to modify the requested bundle name here.
+        #
+        # If we are loading replay, do not add the es5 modifier, as those bundles are
+        # ES6 only.
+        if sdk_version >= Version("7.0.0") and not has_replay:
```

Comment:
Hmm, just thinking, do we need to handle this somehow, as replay generally does not work before 7?